device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/DeviceCalendarPlugin.kt

```diff
@@ -25,10 +26,14 @@ class DeviceCalendarPlugin() : MethodCallHandler {
     val RETRIEVE_CALENDARS_METHOD = "retrieveCalendars";
     val RETRIEVE_CALENDAR_EVENTS_METHOD = "retrieveEvents";
```

Comment:
i think we should remove _CALENDAR_ from there since the Dart method is retrieveEvents not retrieveCalendarEvents

Reply:
Can do